### PR TITLE
chore: allow Nx Cloud read-only token

### PR DESCRIPTION
**Pull request by LayZeeDK**
## Description
Attempt to allow the Nx Cloud read-only access token in GitGuardian.

## References

-

### CODE REVIEW

**Comment by LayZeeDK** [comment]
Need GitGuardian API key.

File: .github/workflows/gitguardian-ignore.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - uses: actions/checkout@v2
+
+      - name: Install package dependencies
+        run: pip install ggshield
+
+      - name: Scan for exposed secrets
+        run: ggshield scan ci
+
+      - name: Ignore last found secret
+        run: ggshield ignore --last-found
```
Comment (dzhavat) [inline .github/workflows/gitguardian-ignore.yml:22]: Question: Wouldn't this make it possible to expose real secrets without them being caught on time by a developer?

Say a developer opens a PR with a secret that must not be exposed. The pipeline will find the secret, add it to the `.gitguardian.yaml` file and commit the file in the repo. Furthermore, the "GitGuardian Security Checks" check will probably raise a warning but that is not so obvious. So we'll have to rely on catching the secret manually during PR reviews, which not might start until after several hours. 

If you think this is unlikely scenario, please ignore the previous section.

Question: If the previous scenario is likely to happen, should the process of ignoring secrets be manual instead? So someone is required to take an intentional action in order to let a secret stay in the repo.

Comment (LayZeeDK) [reply .github/workflows/gitguardian-ignore.yml:22]: This is a manually triggered workflow. I'm trying to figure out the right configuration entry format but we can't do anything without an API key.

**Comment by frontend-specialisten** [comment]
@LayZeeDK Is this still relevant?

**Comment by LayZeeDK** [comment]
@frontend-specialisten
I don't see GitGuardian status checks failing anymore so we should be able to close it. I'll keep the branch for now.

**Pull request closed without merging**